src/components/skeletonView/index.tsx

```diff
@@ -190,8 +180,18 @@ class SkeletonView extends Component<InternalSkeletonViewProps, SkeletonState> {
   static contentTypes: typeof ContentType = ContentType;
 
   fadeInAnimation?: Animated.CompositeAnimation;
+  contentAccessibilityProps?: any;
+  listItemAccessibilityProps?: any;
 
-  constructor(props: InternalSkeletonViewProps) {
+  getAccessibilityProps(accessibilityLabel: string, accessibilityProps: any) {
```

Comment:
In #2154 PR, there is a performance fix by removing accessibility prop when its not needed. Is that not relevant here?

Reply:
I agree, we should optimize how and when to add accessibility props. 
We'll do this in a dedicated PR

Reply:
It's a simple check if `Constants.accessibility.isScreenReaderEnabled`

Reply:
I know, but it's better to do it in a single place instead of having this check in all of our components. 
We should this once here -> https://github.com/wix/react-native-ui-lib/blob/faead14c8f9bc2c56753ac26af79fee4caf36ad7/src/commons/modifiers.ts#L264

Reply:
It should be in the `getAccessibilityProps`, not all components use `extractAccessibilityProps` when they pass the props...

Reply:
Looking at the code again, it looks like we don't need `extractAccessibilityProps` here as well (`props` are destructured afterwards anyway).
In regards to adding `isScreenReaderEnabled` I seem to remember 'accessibilityLabel' was used in some tests\testing env, not sure it is still true.

Reply:
You are right, I believe in some cases we use accessibility props in our tests 
In any case, let's handle this separately from this PR. 

Once we'll figure out a solution we'll create a new PR, I want to push forward this PR